notes/v1.4.7.md

```diff
@@ -0,0 +1,56 @@
+# bloop `v1.4.7`
+
+Bloop v1.4.7 is a bugfix release.
+
+## Installing Bloop
+
+For more details about installing Bloop, please see [Bloop's Installation Guide][install-guide]
+
+## Merged pull requests
+
+Here's a list of pull requests that were merged:
+
+|- Handle jvmOptions without -J prefix in debugSession/start request [#1455]
```

Comment:
What is this `|` char at the beginning of each line? It doesn't seem to be formatted correctly by github: https://github.com/tgodzik/bloop/blob/release-notes-1.4.7/notes/v1.4.7.md

Reply:
Och no, I trusted my old script and it seems that I still have bug. Good catch.

Reply:
Fixed!